sdk/src/azure/core/az_log.c

```diff
@@ -16,75 +16,43 @@
 
 #ifndef AZ_NO_LOGGING
 
-static az_log_classification const* volatile _az_log_classifications = NULL;
+// Only using volatile here, not for thread safety, but so that the compiler does not optimize what
```

Comment:
If you want to keep this cmment, place it above, not between volatiles.

Reply:
It was above...